Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM golang:1.12-alpine as builder
+
+# Install SSL ca certificates
+RUN apk update && apk add git && apk add ca-certificates
+
+# The working directory should be /opt/test-runner
+WORKDIR /opt/test-runner
+RUN mkdir /opt/test-runner/output
+
+# Create appuser
+RUN adduser -D -g '' appuser
+RUN mkdir /home/appuser/.cache/go-build -p
+
+
+# source code
+COPY ./cmd/testoutput/main.go /opt/test-runner/main.go
+COPY ./go.mod /opt/test-runner/go.mod
+COPY ./go.sum /opt/test-runner/go.sum
+
+COPY ./exreport /opt/test-runner/exreport
```

Comment:
I wouldn't change the structure. You can just copy the entire `.` folder in. This container is a throwaway anyway, since the actual container is create below from line 42